README.md

```diff
@@ -1,2 +1,29 @@
-# CC-Global-Summit
-Website made in base of Cindy Shin's design posted in behance
+##Yearly World Comedian Conference
+
+In this website you will be able to see all the information regarding to the Yearly World Comedian Conference, from the date and place it's gonna be done this year to our logo's origin.
+
+
+## Built With
+- HTML & CSS
+- Javascript
+- GitHub
+
+Getting Started
+At this point, you can clone or download the code, open the index.html file and you'd be able to open the website in your browser.
+
+👤 Author1
+
+  GitHub: @Oscar-Espinoza
+  LinkedIn: https://www.linkedin.com/in/oscar-espinoza-68a398a8
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+## Watch my walkthrough video for the website
+
+Loom video link:
```

Comment:
- [ ] Kindly use the new Microverse template for the readme file as this is the latest requirement. [Follow this link](https://github.com/microverseinc/readme-template/blob/master/README.md)